### PR TITLE
feat: Implement user login

### PR DESCRIPTION
**Pull request by Olatisunkanmi**
1. Implemented the user login feature

### CODE REVIEW

File: app/Http/Controllers/v1/Auth/LoginController.php [hunk 1]
```diff
@@ -3,16 +3,35 @@
...
 {
     // Login Route
     public function login() : JsonResponse 
-    {
-        return response()->json([
-            'message' => 'User Logged in'
-        ], 200);
+    {   
+        try {
+            $user = User::where('email', request('email'))->first();
```
Comment (davydocsurg) [inline app/Http/Controllers/v1/Auth/LoginController.php:17]: Please, validate incoming data

File: routes/api.php [hunk 2]
```diff
@@ -14,9 +15,15 @@
 |
 */
```
Comment (davydocsurg) [inline routes/api.php:17]: there's no need to redefine the login routes, it's already defined in `routes/auth.php`

**Review by davydocsurg — CHANGES_REQUESTED** [review]

File: routes/api.php (hunk 2 above)
Comment (Olatisunkanmi) [reply routes/api.php:17]: Can you explain this better, please

Comment (davydocsurg) [reply routes/api.php:17]: Okay, check the `routes/auth.php`. You'll see that the auth routes have already been defined there and then imported in `routes/api.php`

Comment (Olatisunkanmi) [reply routes/api.php:17]: Please check again, pushed a new commit

**Review by davydocsurg — APPROVED** [review]

**Pull request merged**